src/mjolnir/ingest_transit.cc

```diff
@@ -672,12 +727,18 @@ void ingest_tiles(const boost::property_tree::ptree& pt,
 }
 
 // connect the stop_pairs that span multiple tiles by processing dangling tiles
-void stitch_tiles(const boost::property_tree::ptree& pt,
+void stitch_tiles(const std::string& transit_dir,
                   const std::unordered_set<GraphId>& all_tiles,
                   std::list<GraphId>& tiles,
                   std::mutex& lock) {
   auto grid = TileHierarchy::GetTransitLevel().tiles;
-  auto transit_dir = pt.get<std::string>("mjolnir.transit_dir");
+
+  // for a missing stop_pair member's onestop_id we had to take the stop's gtfs stop_id
+  // but generated nodes actually have their node_type appended on the onestop_id, so here we
+  // need to remove the node_type before matching a candidate tile's nodes with the missing ones
+  const auto stop_id_from_onestop_id = [](const auto& n_oid, NodeType node_type, bool is_generated) {
```

Comment:
so one interesting question i had about this was that when we do the stitching itself and we are robust to this name not having the _type on the end, we dont actually patch the onestop_id when we find out if it is actually generated in the other tile. we talked about it and it sounds like we never use the id again so its safe but itd be good to write that in this comment because i think its a pretty tricky but important caveat. in other words if we ever decided we needed the one stop id again (for deduplication for example, it would be important to update the id as well to match its generated counter part, maybe, maybe it still wont be important but good to know about)